### ISSUE TITLE
No output?

### ISSUE THREAD

**Issue by nk9** [stephanvermeire/phaser-bitmapfont-generator#5]
Hi Stephan,

When I run the tool, I get no output. The script runs without errors, but it doesn't seem to do anything. Am I doing something wrong?

I'm on macOS 10.14.6 with npm 6.14.9.

```sh
⋊> ~/ $ mkdir makefont
⋊> ~/ $ cd makefont
⋊> ~/makefont $ npm init -y
Wrote to ~/makefont/package.json:

{
  "name": "makefont",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC"
}


⋊> ~/makefont $ npm install @rtpa/phaser-bitmapfont-generator
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported

> canvas@2.6.1 install /Users/nick/makefont/node_modules/canvas
> node-pre-gyp install --fallback-to-build

node-pre-gyp WARN Using request for node-pre-gyp https download 
[canvas] Success: "/Users/nick/makefont/node_modules/canvas/build/Release/canvas.node" is installed via remote

> core-js@3.8.2 postinstall /Users/nick/makefont/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


> pngquant-bin@5.0.2 postinstall /Users/nick/makefont/node_modules/pngquant-bin
> node lib/install.js

  ✔ pngquant pre-build test passed successfully
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN imports-loader@1.2.0 requires a peer of webpack@^4.0.0 || ^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN exports-loader@1.1.1 requires a peer of webpack@^4.0.0 || ^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN makefont@1.0.0 No description
npm WARN makefont@1.0.0 No repository field.

+ @rtpa/phaser-bitmapfont-generator@1.2.1
added 503 packages from 385 contributors and audited 503 packages in 59.178s

20 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

⋊> ~/makefont $ tee index.js
const generator = require('@rtpa/phaser-bitmapfont-generator');

(async ()=>{

    //generate textures
    await generator.TextStyle2BitmapFont();

    //exit node
    return process.exit(1);

})();const generator = require('@rtpa/phaser-bitmapfont-generator');

(async ()=>{

    //generate textures
    await generator.TextStyle2BitmapFont();

    //exit node
    return process.exit(1);


})();
⋊> ~/makefont $ node .
⋊> ~/makefont $ ls -la
total 312
drwxr-xr-x    6 staff   192B  4 Jan 22:21 ./
drwxr-xr-x@   8 staff   256B  4 Jan 22:19 ../
-rw-r--r--    1 staff   199B  4 Jan 22:21 index.js
drwxr-xr-x  408 staff    13K  4 Jan 22:21 node_modules/
-rw-r--r--    1 staff   147K  4 Jan 22:21 package-lock.json
-rw-r--r--    1 staff   297B  4 Jan 22:21 package.json
```

**Comment by nk9** [comment]
It seems that Phaser might be required as well, so I installed that too:

```
$ npm install phaser
$ cat package.json
{
  "name": "makefont",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "phaser": "^3.50.1"
  },
  "devDependencies": {
    "@rtpa/phaser-bitmapfont-generator": "^1.2.1"
  }
}
```

Still nothing, though.

**Comment by tamagokun** [comment]
same, seems to run, but no output. Verified that the scene's `create` method is not even getting run, so seems to be something up with Phaser internals.

**Comment by tamagokun** [comment]
I got this working by using an older version of Phaser. However, since this library has phaser listed as a dependency with `^` I had to specify the exact version using resolutions:

```json
{
  "dependencies": {
    "@rtpa/phaser-bitmapfont-generator": "^1.2.1",
    "phaser": "3.22.0"
  },
  "resolutions": {
    "phaser": "3.22.0"
  }
}
```

**Comment by stephanvermeire** [comment]
thank you for your finding and feedback.  I am indeed still working in Phaser v.3.22 and it will probably take a while for me to upgrade all my projects. I will have a look at it when I bump to Phaser v.3.5

**Comment by stephanvermeire** [comment]
The version of Phaser had been fixed in the dependencies to 3.22.0. I think this will work for now.
